src/views/world/view_world.cpp

```diff
@@ -137,6 +137,17 @@ namespace big
 
 				deleting = false;
 			});
+
+			ImGui::SeparatorText("Miscellaneous");
+
+			int override_waves_as_int = static_cast<int>(g.world.override_waves);
+			if (ImGui::SliderInt("Waves strength", &override_waves_as_int, 0, 5))
+			{
+				g.world.override_waves = static_cast<float>(override_waves_as_int);
+				MISC::WATER_OVERRIDE_SET_STRENGTH(g.world.override_waves);
```

Comment:
Executing a native in the UI thread, this should be done in a fiber.

Reply:
I don't know what you mean

Reply:
> I don't know what you mean

Which is exactly why this PR is gonna be rejected and you need to make this a Lua instead.